modules/Bio/EnsEMBL/Variation/Pipeline/AncestralAlleles/Init.pm

```diff
@@ -107,11 +107,17 @@ sub store_previous_release_stats {
   my $self = shift;
   my $species_name = shift;
   my $species_dir = shift;
+  my $non_dbSNP_only = shift;
+  my $sql = qq/SELECT ancestral_allele, COUNT(*) FROM variation_feature GROUP BY ancestral_allele;/;
+  if ($non_dbSNP_only) {
+    $sql = qq/SELECT ancestral_allele, COUNT(*) FROM variation_feature WHERE source_id != 1 GROUP BY ancestral_allele;/;
```

Comment:
In nomascus_leucogenys_variation_104_3, dbSNP source_id = 2.
Maybe it would be better to fetch the source id.